### PR TITLE
Problem: need to manipulate with routing table

### PR DESCRIPTION
**Pull request by vyskocilm**
Solution: see commits

### CODE REVIEW

File: tools/fty-route [hunk 1]
```diff
@@ -42,43 +65,58 @@ if [[ -z "${IFACE_PATH}" ]]; then
...
+        fty-route del -net 10.41.0.0/16 gw 192.168.0.1
 
-#augtool -S -I/usr/share/fty/lenses/ rm /files/etc/network/interfaces/iface[4]/post-up[1]
+        Remove route for specific device
+        fty-route del -net 10.41.0.0/16 gw 192.168.0.1 LAN3
+EOF
+        exit 1
+        ;;
+    *)
+        die "Unknown command ${ARGS[0]}, use add/del"
```
Comment (jimklimov) [inline tools/fty-route]: Need `;;` here ;)

File: docs/examples/sudoers.d/fty_00_base.in [hunk 2]
```diff
@@ -34,10 +34,10 @@ Cmnd_Alias BIOS_LOGHOST_RSYSLOG   = @libexecdir@/@PACKAGE@/loghost-rsyslog
 Cmnd_Alias BIOS_MYSQLDUMP   = /usr/bin/mysqldump
 Cmnd_Alias BIOS_DMF         = /usr/bin/bios-dmf
 Cmnd_Alias BIN_SYSTEMCTL    = /bin/systemctl
-Cmnd_Alias BIN_IP_ROUTE     = /bin/ip route
+Cmnd_Alias BIN_FTY_ROUTE    = @libexecdir@/@PACKAGE@/fty-route
```
Comment (jimklimov) [inline docs/examples/sudoers.d/fty_00_base.in]: Not sure we need this at all - the script does `sudo` for commands that need elevation and are already allowed.

Comment (jimklimov) [inline docs/examples/sudoers.d/fty_00_base.in]: Also, in the spirit of variables named above, although clearly not done everywhere "cleanly", `BIN_` prefix is for OS-provided tools, and `BIOS_` (or maybe `FTY_`) is for wrappers and tools from our project, so these differences are better visible for maintenance etc. (e.g. to estimate "is it our direct failure that tool X does not work as expected?").

File: tools/fty-route (hunk 1 above)
Comment (vyskocilm) [inline tools/fty-route]: done

File: tools/fty-route [hunk 3]
```diff
@@ -70,8 +70,8 @@ case ${ARGS[0]} in
         /sbin/route
         ;;
     add)
+        route "${@}" || die "Command route ${@} failed"
```
Comment (jimklimov) [inline tools/fty-route]: Ok, this approach does need `fty-route` in `sudoers` :)

**Pull request merged**